dspace-api/src/main/java/org/dspace/embargo/EmbargoServiceImpl.java

```diff
@@ -142,9 +142,9 @@ public DCDate getEmbargoTermsAsDate(Context context, Item item)
         // sanity check: do not allow an embargo lift date in the past.
         if (liftDate.before(new Date()))
         {
-            throw new IllegalArgumentException(
-                    "Embargo lift date must be in the future, but this is in the past: "
-                            + result.toString());
+            log.info("Embargo lift date must be in the future, but this is in the past");
+            //DS-3321, existing code assumes a null lift date when in the past
```

Comment:
Not sure I understand the context behind this change and the comment.  In 5.x, `EmbargoManager.getEmbargoTermsAsDate()` also threw this `IllegalArgumentException` if a date was in the past, see: https://github.com/DSpace/DSpace/blob/dspace-5_x/dspace-api/src/main/java/org/dspace/embargo/EmbargoManager.java#L154

This makes me slightly suspicious that something else is going on here.  The code logic in *both* `getEmbargoTermsAsDate()` AND `setEmbargo()` appears nearly identical between 5.x and master.  

It is possible there was some "hidden" bug in this logic that we never encountered in 5.x (but we are encountering now in the refactored API).  But, I'd like us to better understand what is going on  here, so we can say with certainty that this exception should *not* be thrown in this scenario.